src/Controllers/PaymentsForwardingController.php

```diff
@@ -28,9 +29,13 @@ public function getCallback(Request $request)
      */
     public function postCallback(Request $request)
     {
-        $invoice = $request->get('invoice');
+        $BitapsTransaction = new BitapsTransaction;
+        $BitapsTransaction->newTransaction($request->all());
 
-        echo $invoice;
-        return response()->noContent();
+        return $this::sendResponse($request->input('invoice'));
+    }
+
+    protected function sendResponse($invoice) {
```

Comment:
@ert-warol, add a type of `$invoice` param. Also, add some comments on that method - what is it for?